campaign-launcher/client/src/components/Account/index.tsx

```diff
@@ -61,8 +63,13 @@ const Account: FC = () => {
     navigate('/manage-api-keys');
   };
 
-  const handleSignIn = () => {
-    signIn();
+  const handleSignIn = async () => {
+    try {
+      await signIn();
+    } catch (error) {
+      console.error(error);
```

Comment:
nit: we already log it in `signIn` itself, will be noisy to get many logs, so would be nice to have just in one
```suggestion
```